packages/xo-server/src/xapi/mixins/storage.js

```diff
@@ -84,4 +84,28 @@ export default {
     })
     return unhealthyVdis
   },
+
+  async createSr({
+    host,
+    nameLabel,
+    nameDescription,
```

Comment:
I know current code is far from clear on this point but try to keep the same field names than the ones used in the `XAPI`.

The idea is that it can be used to (almost) clone an object:

```js
await xapi.createSr({
  ...existingSs,
  name_label: 'clone',
})
```

I've started using this with `createVdi` and `createVbd` and it's very handy.